test/test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
did you add a dedicated test ?
And did you test yourself the error use case (loading the setupChat in head) ?  If so what does happen ?

Reply:
I have added test case in the same file, since it is not working out I have removed it.
Yeah I have tested the error use case, I got the same error appendChild of null.